src/linux/vnode.c

```diff
@@ -61,13 +61,13 @@ inotify_event_dump(struct inotify_event *evt)
 
 /* TODO: USE this to get events with name field */
 int
-get_one_event(struct inotify_event *dst, int inofd)
+get_one_event(char* buf, int buf_len, int inofd)
```

Comment:
Should be uint8_t and size_t.  size_t definitely as that's what read takes.  surprised the compiler didn't complain.